samples/tapi/hog.cpp

```diff
@@ -117,7 +115,7 @@ App::App(CommandLineParser& cmd)
          << "\t4/r - increase/decrease hit threshold\n"
          << endl;
 
-    make_gray = cmd.has("gray");
+    make_gray = cmd.get<bool>("gray");
```

Comment:
make_gray will always be true here without the change. The change is needed so that the command line arguments actually affect something.